monkey/monkey_island/cc/ui/src/styles/Collapse.scss

```diff
@@ -50,8 +51,12 @@ $warning-color: #ffe28d;
   background-color: $info-color !important;
 }
 
-.collapse-default {
-  background-color: $default-color !important;
```

Comment:
We should keep default so that the report doesn't crash on other people's forks/tests of Monkey